### DISCUSSION TITLE
[Feature request]: Issue with reading .sav file on visualboyadvance-m-Win-x86_64

### DISCUSSION THREAD

**Discussion by Origamer22** [visualboyadvance-m/visualboyadvance-m#1363] [General]
### Request box

Hi there.
I saw in one of your issue guides that you managed to help someone play the game they saved from one GBA emulator to another.
I have the same issue. I copied Pokemon Green Leaf from My Boy Emulator on Android (- the way was: Going into Files/.../My Boy/ and copying the .zip and the .sav (that was saved last - when I last saved it on the "My Boy Emulator"), then copying it to my PC, putting them both in the same folder, and the booting up visualboyadvance-m-Win-x86_64 with the game via: "File" > "Open Game Boy..." / "Open Game Boy Color..." , picking the right .zip folder, and it shows me that I'm still on the first 5 minutes of the game.

### Reason box

What do I do?

### Examples

As broght above.

**Comment by Squall-Leonhart** [comment]
you're using a save that you've imported onto the device, not the save that myboy is using.

refer to https://www.reddit.com/r/EmulationOnAndroid/comments/138ufnf/android_update_myboy_cant_access_saves/jj1zc2c/

the srm, fla/frm and eep files may then need to be converted https://savefileconverter.com/#/mister
